pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/mutations/UploadErrorsNotificationSubscriptionMutationService.kt

```diff
@@ -0,0 +1,146 @@
+package gov.cdc.ocio.processingstatusapi.mutations
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.server.operations.Mutation
+import gov.cdc.ocio.processingstatusapi.mutations.models.NotificationSubscriptionResult
+import gov.cdc.ocio.processingstatusapi.mutations.response.SubscriptionResponse
+import io.ktor.client.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.plugins.logging.*
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.Serializable
+
+
+/**
+ * Upload errors subscription data class which is serialized back and forth which is in turn subscribed in to the MemCache
+ * @param dataStreamId String
+ * @param dataStreamRoute String
+ * @param jurisdiction String
+ * @param daysToRun List<String>
+ * @param deliveryReference String
```

Comment:
What is the delivery reference?  We may want to give a description here or rename.  I think the original intent of the delivery reference was to be a reusable delivery reference.  In other words, you setup a delivery for emails to a list of people and could reuse this across different subscriptions.  If we haven't done that and this is just the delivery type or something else, we should explain that here and/or rename the variable.